build/ci/jenkins_pipeline_build

```diff
@@ -1,4 +1,11 @@
 pipeline {
+    parameters {
+        string(name: 'IMAGE_VERSION', defaultValue: "1.7.0")
+        string(name: 'OPERATOR_IMAGE', defaultValue: "ibm-block-csi-operator-amd64")
+        string(name: 'DOCKER_REGISTRY', defaultValue: "stg-artifactory.haifa.ibm.com:5030")
+        string(name: 'EMAIL_TO', defaultValue: "")
+        string(name: 'GOSEC_EXCLUDE', defaultValue: "production", description: "comma separated list of rules to exclude.\nRules are listed here: https://github.com/securego/gosec")
```

Comment:
where does the `production` value come from? is it new? what does it mean?

Reply:
again, I think this parameter can also be removed, since it [doesn't belong here](https://github.com/IBM/ibm-block-csi-driver/pull/354#discussion_r685867970)

Reply:
remove it from both the build job and this here jenkinsfile

Reply:
done